source/Calamari/Kubernetes/Commands/KubernetesDeploymentCommandBase.cs

```diff
@@ -80,36 +84,36 @@ public override int Execute(string[] commandLineArguments)
             {
                 new DelegateInstallConvention(d =>
                 {
+                    var workingDirectory = d.CurrentDirectory;
+                    var stagingDirectory = Path.Combine(workingDirectory, StagingDirectoryPath);
+                    var packageDirectory = Path.Combine(stagingDirectory, PackageDirectoryPath);
```

Comment:
How comfortable are we that this won't clash with any other folder called `package` in `staging`? Could the user create this folder or could extracting an actual package go here?

I'm just concerned with the lack of uniqueness (like we have for the working directory)...

Reply:
I don't think users have any control over the contents of the staging directory, it's just a place where Calamari stores files and things for processing the specific command. The unique working directory is required as if a Calamari process fails to clean-up correctly, we want to make sure we don't have clashes.

Reply:
ok, sounds good to me